build.gradle

```diff
@@ -9,6 +9,7 @@ buildscript {
         opensearch_version = System.getProperty("opensearch.version", "3.0.0-SNAPSHOT")
         spring_version = "5.3.22"
         jackson_version = "2.13.4"
```

Comment:
Do we still need jackson_version?

Reply:
Yes, it is used in other Jackson dependencies such as `jackson-core` , `jackson-annotations`, `jackson-dataformat-cbor`